File: TransCoder_tokenized_test_set_functions/python/MINIMUM_OPERATIONS_REQUIRED_SET_ELEMENTS_BINARY_MATRIX.py
def minOperation ( arr ) :
    ans = 0
    for i in range ( N - 1, - 1, - 1 ) :
        for j in range ( M - 1, - 1, - 1 ) :
            if ( arr [ i ] [ j ] == 0 ) :
                ans += 1
                for k in range ( i + 1 ) :
                    for h in range ( j + 1 ) :
                        if ( arr [ k ] [ h ] == 1 ) :
                            arr [ k ] [ h ] = 0
                        else :
                            arr [ k ] [ h ] = 1
    return ans

File: TransCoder_tokenized_test_set_functions/cpp/MINIMUM_OPERATIONS_REQUIRED_SET_ELEMENTS_BINARY_MATRIX.cpp
int minOperation ( bool arr [ N ] [ M ] ) {
  int ans = 0;
  for ( int i = N - 1;
  i >= 0;
  i -- ) {
    for ( int j = M - 1;
    j >= 0;
    j -- ) {
      if ( arr [ i ] [ j ] == 0 ) {
        ans ++;
        for ( int k = 0;
        k <= i;
        k ++ ) {
          for ( int h = 0;
          h <= j;
          h ++ ) {
            if ( arr [ k ] [ h ] == 1 ) arr [ k ] [ h ] = 0;
            else arr [ k ] [ h ] = 1;
          }
        }
      }
    }
  }
  return ans;
}